lib/mixin/typer/Typer.hx

```diff
@@ -99,7 +99,10 @@ class Typer
 				case IAsName(alias): 
 					addImport(subModule, alias);					
 					
-				case IAll: throw 'Wildcard imports are not supported in mixins';
+				case IAll:
+					// TODO: Is this catch really necessary?
+					//throw 'Wildcard imports are not supported in mixins';
```

Comment:
Having this check makes my project not compile, though I have no Wildcard imports within the module defining the @mixin interface, so I removed it. Are there any side effects to this?

Reply:
Well, yes, there will be side effects, Typer.hx will be unable to resolve any types under wildcard imports, since type resolution done "manually".